app_dart/lib/src/model/luci/buildbucket.dart

```diff
@@ -126,6 +127,9 @@ class Response extends JsonBody {
   /// The [Build] response corresponding to a cancelBuild request.
   final Build cancelBuild;
 
+  /// Error code of the unsuccessful request.
+  final int error;
```

Comment:
From the referenced link: `// Error code and details of the unsuccessful RPC.`. Is it the type of error `int`?

Reply:
https://source.corp.google.com/piper///depot/google3/google/rpc/status.proto?q=google.rpc.Status